torrt/notifiers/telegram.py

```diff
@@ -32,11 +32,13 @@ def test_configuration(self):
 
     def send_message(self, msg):
         url = '%s%s/sendMessage' % (self.url, self.token)
-        r = requests.post(url, data={'chat_id': self.chat_id, 'text': msg})
-        if r.json()['ok']:
-            LOGGER.info('Telegram message was sent to user %s' % self.chat_id)
-        else:
-            LOGGER.error('Telegram notification not send: %s' % r['description'])
+        response = requests.post(url, data={'chat_id': self.chat_id, 'text': msg})
+        if response.ok:
```

Comment:
Wouldn't it better to raise an exception (so to have a chance to further investigate cause of an issue) with ``raise_for_status``?